templates/components/products/modals/writeReview.html

```diff
@@ -44,17 +44,17 @@ <h5 class="product-title">{{ product.title }}</h5>
 
                 <!-- Name -->
                 {{#if product.reviews.author}}
-                    {{> components/common/forms/text id="revfromname" name="revfromname" label=(lang 'products.reviews.form_write.name') value=product.reviews.author}}
+                    {{> components/common/forms/text id="revfromname" name="revfromname" required="true" label=(lang 'products.reviews.form_write.name') value=product.reviews.author}}
```

Comment:
Maybe here we also need to use aria-required instead of simple required?

Reply:
I'm passing required='true' to partial element which will be converted to aria-required='true'